Src/BootCamp.Chapter/ArgumentsHandler.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.IO;
+
+namespace BootCamp.Chapter
+{
+    internal class ArgumentsHandler
+    {
+        private const int MinimumArgumentCount = 2;
+        private const int InputArgumentIndex = 0;
+        private const int CommandArgumentIndex = 1;
+
+        public static ArgumentsHandler Create(string[] args)
```

Comment:
It's not clear what does ArgumentsHandler.Create means. I think a better name is ``Initialize``.